src/routes/user.js

```diff
@@ -20,4 +20,20 @@ router.post('/', async (req, res) => {
   });
 });
 
+router.post('/login', async (req, res) => {
+  const user = new User(
+    {
+      username: req.body.username,
+      password: req.body.password,
+    },
+  );
+  user.findOne({ username: user.username, password: user.password }).then(existingUser => {
+    if (!existingUser) {
+      // the error should be redone, probably with some error library
+      return res.status(404).json({ usernotfound: 'User not found' });
+    }
+    return res.send(existingUser);
+  });
```

Comment:
I would prefer this was an instance method on `User` ie `User.register()`, especially since we'll be adding extra functionality like hashing later.

Reply:
It actually may be more beneficial to use a static method instead of an instance method on `User`

Reply:
This is for logging in, not registering.

Reply:
> This is for logging in, not registering.

I updated my comment

Reply:
> I would prefer this was an instance method on `User` ie `User.login()`, especially since we'll be adding extra functionality like hashing later.

Where do you think I should add that method? In the user model or in the user route?

Reply:
> Where do you think I should add that method? In the user model or in the user route?

The model:
https://mongoosejs.com/docs/guide.html#statics

Reply:
> The model:
> https://mongoosejs.com/docs/guide.html#statics

Thank you, I'll add it soon.

Reply:
> > I would prefer this was an instance method on `User` ie `User.login()`, especially since we'll be adding extra functionality like hashing later.
> 
> Where do you think I should add that method? In the user model or in the user route?

The implementation would look similar to the `saveUser` function found in [models/user.js](https://github.com/Imageus-OSS/server/blob/c6144b8ec4a705339e3354775aa415102b6445b3/src/models/user.js#L14)